avaframe/com1DFAPy/com1DFA.py

```diff
@@ -468,6 +468,10 @@ def initializeSimulation(cfg, demOri, releaseLine, entLine, resLine, logName, re
     cfgGen = cfg['GENERAL']
     methodMeshNormal = cfg.getfloat('GENERAL', 'methodMeshNormal')
 
+    # add dem origin to cfg
+    cfgGen['xllcenter'] = str(demOri['header'].xllcenter)
+    cfgGen['yllcenter'] = str(demOri['header'].yllcenter)
```

Comment:
Maybe we can store the hole header from the original dem in the cfg. This way, in exportFields we do not need the whole demOri but just the header from the cfg.

Reply:
yes this sounds good, i adapted this

Reply:
this is problematic as it would have to a string to be appended to the cfg